packages/beta/src/__tests__/api/alertsApi.int.spec.ts

```diff
@@ -201,3 +201,44 @@ test('sort alerts', async () => {
   });
   expect(response.items.length).toBeGreaterThan(0);
 });
+
+test('cursor pagination', async () => {
+  const client: CogniteClient = setupLoggedInClient();
```

Comment:
suggestion(blocking): we already have a client instantiated above at starting of `describe` block.
We can avoid initiating it over here.